opm/autodiff/DuneMatrix.hpp

```diff
@@ -91,7 +91,11 @@ namespace Opm
             nnz = ia[rows];
 
 #if DUNE_VERSION_NEWER(DUNE_ISTL, 2, 3)
+#if DUNE_VERSION_NEWER(DUNE_ISTL, 2, 5)
```

Comment:
I had/have serious troubles with getting the `DUNE_ISTL_VERSION_*` macros reliably generated by the build system. can this be changed to `DUNE_COMMON`? (that one seems to work better for some reason.)